plenum/server/message_req_processor.py

```
@@ -173,12 +173,12 @@ def _validate_requested_preprepare(self, **kwargs):
                     # the input is expected as a dict (serialization with ujson==1.33)
                     pp = PrePrepare(**kwargs['pp'])
                     if pp.instId != kwargs['inst_id'] or pp.viewNo != kwargs['view_no']:
-                        logger.warning('{} found PREPREPARE {} not satisfying '
-                                       'query criteria'.format(self, *kwargs['pp']))
+                        logger.debug('{} found PREPREPARE {} not satisfying '
+                                     'query criteria'.format(self, *kwargs['pp']))
                         return
                     return pp
                 except TypeError as ex:
-                    logger.warning(
```

Let's not hide this exception. I think warning or error should be fine